operator/v1/types_network.go

```diff
@@ -788,3 +815,24 @@ const (
 	// between pods on the cluster network.
 	IPsecModeFull IPsecMode = "Full"
 )
+
+// AdvancedRoutingProvider is a component providing advanced routing capabilities.
+// +kubebuilder:validation:Enum=FRR
+type AdvancedRoutingProvider string
+
+const (
+	// AdvancedRoutingProviderFRR determines FRR is providing advanced routing
+	// capabilities.
+	AdvancedRoutingProviderFRR AdvancedRoutingProvider = "FRR"
```

Comment:
What is FRR? Can you explain it to me in basic terms? Would spelling out the name be more obvious to an end user?

Reply:
Answered in the previous comment.